README.md

```diff
@@ -386,6 +385,54 @@ Private properties should be declared in class extensions (anonymous categories)
 @end
 ```
 
+## Method Ordering
+
+Method implementations are grouped and ordered with lifecycle methods first, then by class inheritance. A `pragma mark -` is used to separate each group.
+
+“Lifecycle methods” are roughly defined, for the purpose of ordering, as those methods dealing with the creation or destruction of instances: initializers, `+alloc`, `-dealloc`, `-initialize`, `-load`, factory methods, `NSCoding` methods, and the like.
```

Comment:
> > > If future methods are added to NSCoding, would they automatically qualify?
>>
> > Yes.
>
>If these hypothetical methods had nothing to do with lifecycle, wouldn’t this also add to the confusion? Same for lifecycle methods that get added to other classes / protocols that aren’t covered.

I find it difficult to consider this particular hypothetical, since `NSCoding` hasn't changed since OS X 10.0 and its documentation seems to tie it quite tightly to object creation and persistence, ie. lifecycle:

> The NSCoding protocol declares the two methods that a class must implement so that instances of that class can be encoded and decoded. This capability provides the basis for archiving (where objects and other structures are stored on disk) and distribution (where objects are copied to different address spaces).